pgdrive/scene_creator/road/road.py

```diff
@@ -29,7 +29,7 @@ def is_negative_road(self):
         return False if self.end_node.find(Road.NEGATIVE_DIR) == -1 else True
 
     def is_valid_road(self):
-        from pgdrive.scene_creator.basic_utils import Decoration
+        from pgdrive.utils.basic_utils import Decoration
```

Comment:
Stop using import inside function

Reply:
Error will be reported without importing from function